config/locales/en.yml

```diff
@@ -2021,7 +2021,6 @@ en:
           charitable_contributions: Made charitable contributions
           charitable_noncash: Charitable contributions (Non-Cash)
           dependent_tax_credit: Dependent tax credit
-          excise_credit: Incarcerated in a county, state, or federal prison for at least 60 days in 2023
```

Comment:
I did a search and did not find any other usages of this key

Reply:
https://github.com/codeforamerica/vita-min/pull/4303